element_calcium_imaging/imaging_no_curation.py

```diff
@@ -298,20 +298,24 @@ class ZDriftMetrics(dj.Computed):
         ProcessingTask (foreign key): Primary key from ProcessingTask.
         ZDriftParamSet (foreign key): Primary key from ZDriftParamSet.
         z_drift (longblob): Amount of drift in microns per frame in Z direction.
+        bad_frames_threshold (int): Drift threshold in microns where frames are excluded from registration.
+        percent_bad_frames (float): Percentage of frames with z-drift exceeding the threshold.
     """
 
     definition = """
     -> ProcessingTask
     ---
     bad_frames=NULL: longblob  # `True` if any value in z_drift > threshold from drift_params.
     z_drift: longblob   # Amount of drift in microns per frame in Z direction.
+    bad_frames_threshold: int  # Drift threshold in microns where frames are excluded from registration.
```

Comment:
why `int`, if it's in micron, this should be `float`

Reply:
```suggestion
    bad_frames_threshold: float  # Drift threshold in microns where frames are deemed as bad frames.
```

Reply:
Let's go with float. I believe the output of the current algorithm is whole number integers but it's a good call future-proof this

Reply:
I'd also reword the `where frames are excluded from registration`
That's not what's happening here, it may/can happen in a subsequent step, but for this ZDrift table, all it's doing is marking those frames as "bad_frames" (nothing about excluding from registration)

Reply:
Yep, good catch. Done.